### ISSUE TITLE
psortb: incorrect number of arguments passed to `sys.exit`

### ISSUE THREAD

**Issue by domgz** [peterjc/pico_galaxy#44]
We got [this bug report on the Galaxy Help forum](https://help.galaxyproject.org/t/psortb-error-on-usegalaxy-eu/12639) for psortb:

```
/bin/sh: line 1: psort: command not found
Traceback (most recent call last):
  File "/opt/galaxy/shed_tools/toolshed.g2.bx.psu.edu/repos/peterjc/tmhmm_and_signalp/e1996f0f4e85/tmhmm_and_signalp/tools/protein_analysis/psortb.py", line 241, in <module>
    sys.exit(
TypeError: exit expected at most 1 argument, got 2

Tool Exit Code 	1
```

Although the true error is `/bin/sh: line 1: psort: command not found` (and we are working on fixing it), as it can be seen above, the mechanism that prints the error to the console is failing.

**Comment by peterjc** [comment]
Ah, fall out from 21348777df4654068d7bc2b04db5296fa89eefe7 - but I'm surprised it took this long to get triggered and reported.

**Comment by peterjc** [comment]
Thank you - I've fixed this slightly differently than #45, but the same idea. Please do get in touch again if there's a similar issue elsewhere.
